Makefile

```diff
@@ -28,31 +32,45 @@ lint:
 	)
 
 test:
-	pytest tests
-
-test-all:
-	tox
+	python -m pytest tests
 
 benchmark:
-	tox -e benchmark
+	python -m tox run -e benchmark
+
+autobuild-docs:
+	sphinx-autobuild --open-browser docs docs/_build/html
 
 build-docs:
-	sphinx-apidoc -o docs/ . setup.py "*conftest*" "tests" "web3/tools/*"
+	sphinx-apidoc -o docs/ . setup.py "*conftest*"
```

Comment:
I think we want to leave this line as-is too. I'm getting a couple warnings that look like: `WARNING: autodoc: failed to import module 'tests'; the following exception was raised:
No module named 'tests'`